be/src/olap/rowset/segment_v2/encoding_info.cpp

```diff
@@ -50,7 +55,7 @@ struct TypeEncodingTraits<type, PLAIN_ENCODING> {
 };
 
 template<FieldType type>
-struct TypeEncodingTraits<type, BINARY_PLAIN_ENCODING> {
+struct TypeEncodingTraits<type, PLAIN_ENCODING, Slice> {
```

Comment:
BINARY_PLAIN_ENCODING and PLAIN_ENCODING is not same.
For HLL or Bitmap type, BINARY_PLAIN_ENCODING is used.

Reply:
In fact there is no need to use different EncodingTypePB to instantiate different PageBuilder for different data type. Partial template specification is used here to create BinaryPlainPageBuilder for type using Slice as CppType.

Reply:
OK, I see